src/main/java/com/limechain/sync/warpsync/state/RuntimeBuildState.java

```diff
@@ -13,7 +11,7 @@
  */
 @Log
 public class RuntimeBuildState implements WarpSyncState {
-    private final SyncedState syncedState = SyncedState.getInstance();
+    protected SyncedState syncedState = SyncedState.getInstance();
```

Comment:
What's the point of this change? `RuntimeBuildState` has no children right?

Reply:
The change is to make the field available for testing.

Reply:
You can keep it private and add

@AllArgsConstructor
@NoArgsConstructor

It will work fine and you will have no warning messages that it's not final.

Reply:
Updated.